src/view/cluster/app/clusterView.tsx

```diff
@@ -0,0 +1,434 @@
+      },
+      '& .MuiStepIcon-root.MuiStepIcon-completed': {
+        color: '#BE0000'
+      },
+      '& .MuiButton-containedPrimary': {
+        backgroundColor: '#BE0000'
+      },
+      '& .MuiStepLabel-iconContainer': {
+        paddingRight: theme.spacing(2)
+      }
+    },
+    button: {
+      marginTop: theme.spacing(1),
+      marginRight: theme.spacing(1)
+    },
+    actionsContainer: {
+      marginBottom: theme.spacing(2),
+      marginTop: theme.spacing(2)
+    },
+    resetContainer: {
+      padding: theme.spacing(3),
+    },
+    formControl: {
+      margin: theme.spacing(1),
+      minWidth: 120,
+      width: '40%'
+    },
+    uploadLabel: {
+      marginTop: theme.spacing(2)
+    },
+    pullSecretField: {
+      marginTop: '20px',
+      width: '100%'
+    }
+  })
+);
+declare global {
+  interface Window {
+      acquireVsCodeApi(): any;
+  }
+}
+
+const vscode = window.acquireVsCodeApi();
+
+const crcVersions = [
+  { crcVersion: "1.0.0", openshiftVersion: "4.2.0"},
+  { crcVersion: "1.1.0", openshiftVersion: "4.2.2"},
+  { crcVersion: "1.2.0", openshiftVersion: "4.2.8"},
+  { crcVersion: "1.3.0", openshiftVersion: "4.2.10"},
+  { crcVersion: "1.4.0", openshiftVersion: "4.2.13"},
+  { crcVersion: "1.5.0", openshiftVersion: "4.2.14"},
+  { crcVersion: "1.6.0", openshiftVersion: "4.3.0"},
+  { crcVersion: "1.7.0", openshiftVersion: "4.3.1"},
+  { crcVersion: "1.8.0", openshiftVersion: "4.3.8"},
+  { crcVersion: "1.9.0", openshiftVersion: "4.3.10"},
+  { crcVersion: "latest", openshiftVersion: "4.4.3"}
+];
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+function getSteps() {
+  return ['Select OpenShift Version', 'CRC Binary Path/Download', 'File path of image pull secret', 'Select optional configurations', 'Start the cluster'];
+}
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+export default function addClusterView() {
+    const classes = useStyles();
+    const [fileName, setBinaryPath] = React.useState('');
+    const [pullSecretPath, setSecret] = React.useState('');
+    const [cpuSize, setCpuSize] = React.useState(4);
+    const [memory, setMemory] = React.useState(8192);
+    const [versionLabel, setVersionLabel] = React.useState('latest');
+    const [crcOut, setOut] = React.useState('');
+    const [crcProgress, setProgress] = React.useState(false);
+    const messageListener = (event) => {
+      if (event?.data?.action){
+        const message = event.data;
+          switch (message.action) {
+            case 'crcoutput' :
+              setOut(message.data);
+              break;
+            case 'crcstatus' :
+              if(message.data === 0) setProgress(true);
+              break;
+            default:
+              break;
+          }
+      }
+    }
+    window.addEventListener('message', messageListener);
+    const crcDownload = `http://mirror.openshift.com/pub/openshift-v4/clients/crc/${versionLabel}/crc-macos-amd64.tar.xz`;
```

Comment:
Always download binaries for macOS.

Reply:
Fixed in commit 83529434748eb4e4fb79ea5fc3e4b5ec13534631